Makefile

```diff
@@ -1,5 +1,6 @@
 CURPATH=$(PWD)
 TARGET_DIR=$(CURPATH)/_output
+KUBECONFIG?=$(HOME)/.kube/config
```

Comment:
Is this necessary?  by default, if KUBECONFIG is not set, the oc command will look in $HOME/.kube/config